controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/role/RoleDefinition.java

```diff
@@ -20,60 +20,46 @@
     /** Deus ex machina. */
     hostedOperator(Policy.operator),
 
+    /** Reader - the base role for all tenant users */
+    publicReader(
+            Policy.tenantRead,
+            Policy.applicationRead,
+            Policy.deploymentRead,
+            Policy.publicRead
+    ),
+
+    /** User - the dev.ops. role for normal Vespa tenant users */
+    publicDeveloper(
+            Policy.applicationCreate,
+            Policy.applicationUpdate,
+            Policy.applicationDelete,
+            Policy.developmentDeployment
+    ),
+
+    /** Admin - the administrative function for user management etc. */
+    publicAdministrator(
+            Policy.tenantUpdate,
+            Policy.tenantManager
```

Comment:
Only we may delete tenants?

Reply:
Missing the `applicationManager` policy.

Reply:
Yes, I think it makes sense that only we delete tenants. I'll look into `applicationManager`.

Reply:
We _could_ remove some granularity in the underlying policies and privileges, but I'm not sure we should, at least not before the new outwards model is set in stone.

Reply:
No, I don't think we should remove granularity there. I'd rather we bucket things together at the top :)